### ISSUE TITLE
the trait quote::Tokens is not implemented for syn::Ident

### ISSUE THREAD

**Issue by giluis** [dtolnay/quote#212]
Hello,
Going through the proc-macro-workshop, Builder project.

While trying to pass the second test case, I got this error:
![image](https://user-images.githubusercontent.com/52404649/159130647-07db8592-84de-4777-9921-a7c689de25bd.png)

The related code is this:

```rust
use proc_macro::TokenStream;
use syn::*;
use quote::*;

#[proc_macro_derive(Builder)]
pub fn derive(input: TokenStream) -> TokenStream {
    let ast = parse_macro_input!(input as DeriveInput);
    let name = ast.ident;
    let expanded = quote!{ #name };
    TokenStream::from(expanded)
}

```
I don't understand what is going on. I have done this in the past with syn and Ident usually implemented ToTokens IIRC

What am I missing?

**Comment by giluis** [comment]
Figured out the problem!
Was using version 1.0.
When I changed to version 1.0.82 it started working. 
Perhaps it would be better to change the cargo.toml section example that is present in the README, which states:
```toml
[dependencies]
quote = "1.0"
```

**Comment by lroux-at-jellysmack** [comment]
Hello @giluis, a `cargo update` would have fixed it while keeping `quote = "1.0"` :)

_PS: No a contributor of `quote`_
